guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,7 +4,20 @@
 [id="foreman-headline-features"]
 == Headline Features
 
-There are no highlights with Foreman {ProjectVersion}.
+=== IPv6 Support
+
+Deployment and operation in IPv6-only networks is now fully supported.
+
+Provisioning over IPv6 is supported on bare metal hosts.
+For compute resources, you can define the machine outside Foreman on the compute resource and then you can provision the machine as bare metal in Foreman.
```

Comment:
Is there any reference on why image mode in IPv6-only wouldn't work?

Reply:
not sure tbh, just what I was given